home_default/templates/home_default/base.html

```diff
@@ -6,6 +6,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" type="text/css" href="{% static 'home_default/css/bootstrap.css' %}" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
```

Comment:
Should these script tags go in the <head>? Afaik, they're supposed to follow the {% endblock content %} tag in the body.

Reply:
In my understanding, it depends on what kind of styling is needed for a given page. I assumed since most of the changes made in the home page should be part of all the other pages, and hence this changes should be loaded into the <head> tag rather than <body>.

Reply:
Yeah - I had added them to the end - during the rebase looks like they moved up

Reply:
I believe the line Mike is asking about is <script> not the style sheet tag - and that's the js portion of it right - so by convention they go to the bottom of the block content

Reply:
Oh! Since the above 2 lines were highlighted for the comment, I thought he meant stylesheet, but wrote the script.

Reply:
Correct- the link to the stylesheet, in my understanding, goes at in the head section and the script tags for bootstrap go at the bottom of the body section.

Reply:
Agreed. Fixing it.